conda_smithy/schema.py

```diff
@@ -864,14 +867,20 @@ class ConfigModel(BaseModel):
         description=cleandoc(
             """
         This key is used to set the OS versions for `linux_*` platforms. Valid entries
-        map a linux platform and arch to either `cos6` or `cos7`.
-        Currently `cos6` is the default for `linux-64`.
-        All other linux architectures use CentOS 7.
-        Here is an example that enables CentOS 7 on `linux-64` builds
+        map a linux platform and arch to either `cos7`, `alma8`, `alma9` or `ubi8`.
+
+        Currently `alma9` is the default, which should work out-of-the-box for the vast
+        majority of uses.
```

Comment:
There isn't really a good place in the pinning to point to, because the logic for the `DEFAULT_LINUX_VERSION` makes it hard to tell. We could point to the knowledge base docs (those need to be kept in sync as well anyway).

OTOH, bumping the default image version is an event that happens every ~4-5 years (going forward at least, as that's roughly the cadence for RHEL releases), so I think we'd manage either way.

Reply:
Ok. We could simply reference the pinnings repo or we could abstract the the default as a templated variable that gets substituted into the pinnings file. Happy to punt on this for now.